man/rgblink.5

```diff
@@ -56,55 +40,105 @@ and
 .Ql \et .
 Other backslash escape sequences in
 .Xr rgbasm 5
-are only relevant to assembly code and do not apply in section names.
+are only relevant to assembly code and do not apply in linker scripts.
+.Ss Directives
+.Bl -tag -width Ds
+.It Including other files
+.Ql Ic INCLUDE Ar path
+acts as if the contents of the file at
+.Ar path
+were copy-pasted in place of the
+.Ic INCLUDE
+directive.
+.Ar path
+must be a string.
+.It Bank specification
+.Tg region
+The active bank can be set by specifying its type (memory region) and number.
+The possible types are:
+.Ic ROM0 , ROMX , VRAM , SRAM , WRAM0 , WRAMX , OAM ,
+and
+.Ic HRAM .
+However, if the type only contains a single bank
+.Pq Ic ROM0, Ic ROMX No if Fl t No is passed to Xr rgblink 1 , Ic VRAM No if Fl d No is passed to Xr rgblink 1 , Ic WRAM0 , Ic WRAMX No if Fl w No is passed to Xr rgblink 1 , Ic OAM , Ic HRAM ,
+then the number can be omitted.
 .Pp
-When a new bank statement is found, sections found after it will be placed right from the beginning of that bank.
-If the linker script switches to a different bank and then comes back to a previous one, it will continue from the last address that was used.
+After a bank specification, the
+.Dq current address
+is set to the last value it had for that bank.
+If the bank has never been active thus far, the
+.Dq current address
+defaults to the beginning of the bank
+.Pq e.g. Ad $4000 No for Ic ROMX No sections .
+.It Changing the Dq current address
+A bank must be active for any of these directives to be used.
 .Pp
-The only three directives are
-.Ic ORG ,
-.Ic ALIGN ,
-and
-.Ic DS :
-.Bl -bullet
-.It
-.Ic ORG Ar addr
-sets the address in which new sections will be placed to
+.Ql Ic ORG Ar addr
+sets the
+.Dq current address
+to
 .Ar addr .
-It can not be lower than the current address.
-.It
-.Ic ALIGN Ar addr
-or
-.Ic ALIGN Ar addr , Ar offset
-will increase the address until it is aligned to the specified boundary
-.Po it tries to set to
+The new address cannot be less than the old one.
+.Pp
+.Ql Ic FLOATING :
+All sections between
+.Ic FLOATING
+and the next
+.Ic ORG
+or bank specification, will be placed at addresses automatically determined by
+.Nm .
```

Comment:
None of the other directives are written as "***Name*:** *Description*".
```suggestion
.Ql Ic FLOATING
causes all sections between
.Ic FLOATING
and the next
.Ic ORG
or bank specification to be placed at addresses automatically determined by
.Nm .
```

Reply:
Well, maybe "between it and the next", then?